features/step_definitions/additional_cli_steps.rb

```diff
@@ -98,7 +98,7 @@
 end
 
 Given /^I have a brand new project with no files$/ do
-  in_current_dir do
+  in_current_directory do
```

Comment:
I think these are why the build is failing BTW, we've upgraded some Aruba things lately